app/retail/templates/idea.html

```diff
@@ -0,0 +1,42 @@
+{% comment %} Copyright (C) 2017 Gitcoin Core This program is free software: you can redistribute it and/or modify it under
```

Comment:
😂 I've seen this happen in VSCode. 
Can we drop the`{% comment %}` block back down the way it normally is?

Reply:
yes, i hit format all the time